README.md

```diff
@@ -2,8 +2,8 @@
 
 [![Built with Cookiecutter Plone Starter](https://img.shields.io/badge/built%20with-Cookiecutter%20Plone%20Starter-0083be.svg?logo=cookiecutter)](https://github.com/collective/cookiecutter-plone-starter/)
 [![Black code style](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/ambv/black)
-[![Backend Tests](https://github.com/plone/plone.org/actions/workflows/backend.yml/badge.svg)](https://github.com/plone/plone.org/actions/workflows/backend.yml)
-[![Frontend Tests](https://github.com/plone/plone.org/actions/workflows/frontend.yml/badge.svg)](https://github.com/plone/plone.org/actions/workflows/frontend.yml)
+[![Backend Tests](https://img.shields.io/badge/Tests-Backend-green)](https://github.com/plone/plone.org/actions/workflows/backend.yml)
```

Comment:
This image will always be green and passing. Also the filename name has changed, which is why it broke.

```suggestion
[![Backend testing.plone.org CI and Deploy](https://github.com/plone/plone.org/actions/workflows/backend-testing.yaml/badge.svg)](https://github.com/plone/plone.org/actions/workflows/backend-testing.yaml)
```